test/passenger_test.rb

```diff
@@ -69,6 +70,65 @@
   end
 
   describe "net_expenditures" do
```

Comment:
The tests for net_expenditures and total_time_spent should test the edge cases of a passenger having no trips an incomplete trips.